plugins/doc_fragments/onepassword.py

```diff
@@ -40,6 +40,8 @@ class ModuleDocFragment(object):
       - The access key for a service account.
       - Only works with 1Password CLI version 2 or later.
     type: str
+    env:
+      - name: OP_SERVICE_ACCOUNT_TOKEN
```

Comment:
```suggestion
```
This must not be added here. `env` is not allowed for modules, which might also use this fragment.